ionic-app/pages/_app.js

```diff
@@ -0,0 +1,31 @@
+import Head from 'next/head';
+import Script from 'next/script';
+
+import 'tailwindcss/tailwind.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
import "tailwindcss/tailwind.css";
```